release/models/isis/openconfig-isis.yang

```diff
@@ -1366,8 +1419,8 @@ module openconfig-isis {
       default false;
       description
         "When this leaf is set to true, authentication of IS-IS PSNP, CSNP and
-        LSP packets is enabled using the authentication details specified in
-        the keychain in the sibling leaf.
+        LSP packets is enabled using the specified authentication details in
+        the sibling leaves.
 
         The simbling 'disable-<type>' leaves can be used to override the value
```

Comment:
`s/simbling/sibling/` - could you fix this whilst making changes to this file please? Thanks!

Reply:
Fixed